scene/gui/text_edit.cpp

```diff
@@ -37,6 +37,7 @@
 #include "core/project_settings.h"
 #include "scene/main/viewport.h"
 
+#include "core/script_language.h"
```

Comment:
Nitpick, but this would be better in the main block of includes above (so on line 38).